### DISCUSSION TITLE
Next version of the DIYson

### DISCUSSION THREAD

**Discussion by LukasHNZ** [stevenbennett/DIYson#45] [General]
Hello, is  there any plans for a pt. 10 of the DIYson series? Or is it considered "completed"?
Kind regards!

**Comment by Joseph0M** [comment]
I was thinking that too! Apparently Steven is on holiday so I am sure when he is back, the repo will have some more attention again. I am hoping for some more community engagement!

**Reply by LukasHNZ** [reply]
Id love to put some work into the DIYson! I need a proper desk lamp and want to build something like that. I think Dyson even discontinued this exact model that inspired it, so now i have one more reason to build it myself.
Is there like a discord for the DIYson or something?

**Reply by Joseph0M** [reply]
A discord or a subreddit was suggested [here](https://github.com/stevenbennett/DIYson/discussions/34), but I think his vision would be a website forum sort of thing

**Reply by LukasHNZ** [reply]
As long as it allows to work and exchange with other people I am all here for it. What do you think of making an inofficial discord/ subreddit and closing it or making it refer to the original page as soon as it is available?

**Reply by Joseph0M** [reply]
I’m probably not the most experienced person when it comes to managing servers, I’m sure others would be more than happy to take a lead, but I would be all in for an idea like this!

**Comment by stevenbennett** [comment]
@LukasHNZ @Joseph0M Hey guys! Apologies for the lack of communication lately, I've been experiencing some burnout on the DIYson project this year and have been recharging with some other projects and hobbies. Thanks for your patience while my contributions to the project are on pause, I really appreciate that. I wanted to let you know that the DIYson is definitely still "in development" in my mind and I plan to make more updates and build log videos in the future, just taking a break for now.

Question for you two: If there was something like an "official" Discord server for this project, what would you expect my involvement to be? Would you be disappointed if I wasn't in there every day chatting and answering questions?

**Reply by Joseph0M** [reply]
Thanks for the little update! I have to admit, I have been struggling with a bit of burnout lately too! (I'm quite enjoying all the electronics for my own PCB prototype.) I think we would all be interested to know about these other projects too. 😄
Anyway, keep up the YouTube thing! Your videos are great, and I love seeing how you create solutions to your problems, especially with your fume extractor.

For an official server, I think we would appreciate some connection to the community, e.g., once or twice a week, but I can appreciate your other commitments, and I wouldn't "expect" your constant involvement or time. Maybe some direction from time to time? We would all definitely appreciate the designs and schematics for things like the super track to increase community engagement and progress our own builds.

If we were to go ahead with the Discord server, we would definitely need a couple elected community members to act as administrators, along with promotion and a short burst of engagement from yourself.

Would you have anything to add to this? @LukasHNZ

**Reply by LukasHNZ** [reply]
Hello @Joseph0M and @stevenbennett ,
Thank you for your reply!

For the first part, I agree with @Joseph0M , i really like your YouTube content, its high quality and entertaining, regardless if i want to make that project myself or not.

For the latter:
I dont really expect anything of you if there is an official discord/ reddit / similar.

You could post updates when a new video is coming into a dedicated channel, and maybe discuss with the community about ideas and technologies, but since we all can exchange with each other aswell there is no "need" for you to constantly participate in the conversations.

Also I can imagine the burnout getting less, with the community working and tinkering together so there will be more progress and faster fixes for problems!

**Reply by stevenbennett** [reply]
@LukasHNZ @Joseph0M Thanks for the thoughtful feedback, I really appreciate it. I'm definitely open to the idea of having an official place for all of us to stay in touch, that sounds fun. I will look into it and let you know 🙂

**Reply by LukasHNZ** [reply]
Hello @stevenbennett any news regarding a discord or similar to exchange for your project?

**Reply by stevenbennett** [reply]
@LukasHNZ Will share the link tomorrow 🙂

**Comment by stevenbennett** [comment]
@LukasHNZ @Joseph0M Alright let's give this a shot and see how it goes. Bear with me for a bit while I figure out how to use Discord properly: https://discord.gg/B43ZVcBme4

**Reply by TheNameIsBart** [reply]
Hey Steven, could you update the invite link? This link seems to be expired
![Screenshot 2023-11-23 at 20-50-44 Discord - A New Way to Chat with Friends   Communities](https://github.com/stevenbennett/DIYson/assets/32821846/a1fc4913-03c5-438b-ac9e-25c306320e4c)

**Reply by Joseph0M** [reply]
Here’s a non-expired link: https://discord.gg/7je2Mhav49

**Reply by LukasHNZ** [reply]
Hi!

https://discord.com/invite/PzhH5GqUUZ

This should be it. Does it work?

**Reply by stevenbennett** [reply]
Thanks @Joseph0M and @LukasHNZ. @TheNameIsBart, I just updated the original post with a link that won't expire.
